README.md

```diff
@@ -5,19 +5,25 @@
 A Gnome extension to show Nvidia GPU information
 (https://extensions.gnome.org/extension/1320/nvidia-gpu-stats-tool/)
 
-# Requirements
-nvidia-settings and nvidia-smi
+## Requirements
+The extension requires `nvidia-smi`, which should be included with the drivers.
+Additionally, `nvidia-settings` is supported as a data source.
```

Comment:
also optimus (i.e. optirun) supported

Reply:
I decided not to advertise Bumblebee support explicitly as I'm unsure if this is still relevant ([the Arch wiki](https://wiki.archlinux.org/title/Bumblebee) disrecommends it). Should I still add a sentence here?

Reply:
You could while also mentioning that as a disclaimer?

Reply:
> I decided not to advertise Bumblebee support explicitly as I'm unsure if this is still relevant ([the Arch wiki](https://wiki.archlinux.org/title/Bumblebee) disrecommends it). Should I still add a sentence here?

You could while also mentioning that as a disclaimer?

Reply:
Try this:
"NOTE: There is also the option to use optimus (i.e. optirun) as a data source - but [this isn't recommended](https://wiki.archlinux.org/title/Bumblebee) and has not been tested with a recent version. The feature will likely be deprecated in a future release."